docs/trestle_author.md

```diff
@@ -18,347 +27,573 @@ While trestle provides editing support for OSCAL there is an unfortunate truth t
+    ### Creating new documents
 
-`trestle author headers validate -tn my_task_name` Will validate all files within the directory against the templates by matching the extensions.
+    As of now this command does not support the automatic creation of the sample documents.
 
-`trestle author headers validate -tn my_task_name -ig ^_.*` will validate all files except folders and files that start with underscore `_`. Use option `-ig` or `--ignore` when you would like to ignore any folders or files that match given regular expression.
+    ### Validating the templates
 
-### Supported options
+    To validate the documents against the template run:
 
-#### Recursive (`-r`, `--recurse`)
+    > trestle author headers template-validate -tn my_task_3
 
-By default `author headers` only indexes a flat directory. The recursive option allows the discovery of sub directories.
+    This will ensure that the respective template files are parseable.
 
-#### Global (`-g`, `--global`)
+    #### Extra options
 
-Allows a single set of templates defined in `.trestle/author/__global__` to be applied to mulitple directories in trestle.
-If `--task-name` is not provided all folders in the repository will be measured.
+    - This command has no extra options
 
-#### Exclude (`-ex`, `--exclude`)
+    ### Validating the documents
 
-Primarily intended for use with global (`-g`), exclude will remove any _directory_ from the search scope of trestle author headers. Must be a relative path to the root of a trestle directory, however, can be multiple levels deep (e.g. `--exclude=architecture/drafts`) would allow content in `architecture/diagrams` to still be indexed to find the header.
+    To validate the documents against the template run:
 
-#### Ignore files or folders from validation (`-ig`, `--ignore`)
+    > trestle author headers validate -tn my_task_3
 
-Provide regular expression after this flag to ignore folders and files which names match this regular expression during **validation only**.
+    This will validate all files within the directory against the templates (of the matching task name) by matching the extensions.
+    Please note that **only** headers will be validated. There is no option to validate the body.
 
-#### Template version tracking (`-tv`, `--template-version`)
+    ### Extra options
 
-This, along with the use of x-trestle-template-version in the governed header allows for repos using author templates
-to implement support for multiple versions of templates. The x-trestle-template-version header represents the version
-of the template used to create an instance document. With this change, the Version header specifically refers to the
-version _of that document_ (be it an instance or the template itself). This means:
+    - If `--global` (`-g`) is passed then `__global__` then it will create `trestle_root/.trestle/author/__global__` folder. Use this when you would like to use same template for multiple tasks.
```

Comment:
I think there is a typo because there are two 'then's.  Not sure what `__global__` is doing.

Reply:
Done